packages/build/src/core/flags.js

```diff
@@ -204,6 +204,11 @@ Default: false`,
     describe: 'Enable distributed tracing for build',
     hidden: true,
   },
+  'tracing.preloadingEnabled': {
+    boolean: true,
+    describe: 'Enable distributed tracing for build via module preloading, to be removed once fully rolled out',
+    hidden: true,
+  },
```

Comment:
This property can be removed once this has been fully rolled out, see:
- https://github.com/netlify/build/blob/21239dca368001072a3cead8805a3c641f744fd5/packages/opentelemetry-sdk-setup/src/sdk-setup.ts#L10-L11